specs/electra/beacon-chain.md

```diff
@@ -70,6 +162,17 @@ The following values are (non-configurable) constants used throughout the specif
 | `MAX_EXECUTION_LAYER_EXITS` | `2**4` (= 16) |  *[New in Electra:EIP7002]* |
```

Comment:
I think we can remove `MAX_EXECUTION_LAYER_EXITS` as it's replaced by `MAX_WITHDRAWAL_REQUESTS_PER_PAYLOAD` in practice.